### ISSUE TITLE
False positive @dataProvider referencing to a non-existing entity.

### ISSUE THREAD

**Issue by Logiar** [kalessil/phpinspectionsea#732]
After the Version: 2.3.15.1 update this gives a false positive. Can reproduce with:
```php
<?php
class Test extends \PHPUnit\Framework\TestCase
{
    /**
     * @dataProvider DataProvider::provideData
     * @param bool $test
     */
    public function testDataProvider(bool $test)
    {
        self::assertTrue($test);
    }
}

class DataProvider
{
    public static function provideData(): array
    {
        return [[true]];
    }
}
```

**Comment by kalessil** [comment]
@Logiar : if you point cursor to `provideData` and keep Ctrl pressed, does IDE allows you to perform navigation?

**Comment by Logiar** [comment]
Yes. It does.

**Comment by kalessil** [comment]
Ok, that's a bug - I'll fix it.

**Comment by kalessil** [comment]
Fixed
